cmsdb/processes/ewk.py

```diff
@@ -339,7 +339,7 @@
 zz_llll = zz.add_process(
     name="zz_llll",
     id=8130,
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={13: Number(1.256)},  # TODO
```

Comment:
Is this the XS of `XS(ZZ)*BR(Z->ll)**2`? Might be good to write this out, or even have a look if e.g. the BR(Z->ll) is already part of the constants

Reply:
Yes, this should be already xsec*BR. If we want to be 1:1 with the main HZZ analysis, we should consider adding the k-factors in the `cf` example to normalize this xsec to the NNLO one, but for the scope of the exercise this should be irrelevant.

Reply:
hi @pkausw done in [this commit](https://github.com/bonanomi/cmsdb/commit/53af1e621008731f8704dcb6f43c5e323b028ec9)